lib/demos/material/picker_demo.dart

```diff
@@ -74,6 +85,19 @@ class _PickerDemoState extends State<PickerDemo> {
     }
   }
 
+  Future<void> _showDateRangePicker() async {
```

Comment:
I believe this will present the full screen dialog over top of the entire gallery. If its not too much to ask, could you rework the demo a bit to have its own `Navigator`, like the dialog demo does. That way the picker dialogs will be displayed over top of the demo area only rather than the whole app. 

See https://github.com/flutter/gallery/blob/master/lib/demos/material/dialog_demo.dart#L127 for an example of what I mean.

Reply:
@rami-a ,Is there any different way to show picker in demo area? ,  I tried adding Navigator in picker demo but it's still showing in fullscreen furthermore and dialougs in dialog demo are all also showing in fullscreen .

Reply:
If you add a navigator and then also pass `useRootNavigator` as false in the `showDateRangePicker` function does that work?

Reply:
@rami-a , Cool it worked, i didn't knew about it , should I make userootnavigator false in all picker?.

Reply:
Great! And yes, please make that change for all pickers so they have similar behavior. Thanks!

Reply:
I've made necessary changes please have a look .